sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsActions.cs

```diff
@@ -60,5 +60,14 @@ public TextAnalyticsActions()
         /// This property only applies for <see cref="TextAnalyticsClientOptions.ServiceVersion.V3_2_Preview_1"/> and up.
```

Comment:
When you delete the `ServiceVersion.V3_2_Preview_1` I expect this docstring to be updated to `ServiceVersion.V3_2_Preview_2`